### DISCUSSION TITLE
help getting telescope bibtex "extension" working

### DISCUSSION THREAD

**Discussion by rhaynes74** [NvChad/NvChad#2526] [Q&A]
Hi folks, I am enjoying nvchad so far.  I am a long time vi/vim user (30+ years now!), but new to nvchad.  I am trying to get the telescope bibtex "extension" working from https://github.com/nvim-telescope/telescope-bibtex.nvim, but I am getting quite lost as to what to put where in the nvchad config ecosystem. I have been able to get all other plugins working but not this one - possibly due to it being a plug extension (which I have no experience with), lazy loading, etc.  Anybody here has this working or are willing to give it a try and report back?   thanks!

**Comment by siduck** [comment]
@rhaynes74 please read our plugin docs

**Reply by rhaynes74** [reply]
I have browsed but without tons of time to study in depth.  The documentation is not terribly long, is there a particular example I am missing that is particularly relevant or are you suggesting the only way is to piece it together piece by piece?

**Reply by siduck** [reply]
simple example for telescope undo plugin, we mention it in extensions_list cuz it'll be loaded by telescope

```lua
{
    "nvim-telescope/telescope.nvim",
    dependencies = { "debugloop/telescope-undo.nvim" },
    opts = {
      extensions_list = { "themes", "terms", "fzf", "undo" },
    },
  }
  ```
  
 the default plugin table of telescope has this code
  
  
![image](https://github.com/NvChad/NvChad/assets/59060246/d0de495f-d22c-4e02-a6bf-f122510aec6d)

**Reply by rhaynes74** [reply]
thanks! this should help.
